AstronomCore/src/main/java/com/astronomvm/core/model/database/DatabaseConnectionProperties.java

```diff
@@ -0,0 +1,13 @@
+package com.astronomvm.core.model.database;
+
+import lombok.Data;
+
+@Data
+public class DatabaseConnectionProperties {
+
+    private String databaseUrl;
+    private JDBCDriver jdbcDriver;
+    private String schemaName;
+    private String username;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "username" private field. ([squid:S1068](https://sonarcloud.io/organizations/ahmedriahi-github/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=AhmedRiahi_AstronomVM&pullRequest=22&issues=AWZfQr2bDZywvPe4Lfui&open=AWZfQr2bDZywvPe4Lfui)

[ISSUE_KEY]: AWZfQr2bDZywvPe4Lfui